osquery/utils/conversions/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ function(generateOsqueryUtilsConversions)
   target_link_libraries(osquery_utils_conversions PUBLIC
     osquery_cxx_settings
     osquery_logger
+    osquery_filesystem
```

Comment:
What is this line?

Reply:
Good call. I think I needed to link the `filesystem` library for this test, but I'll try removing and double check that it's necessary.